### DISCUSSION TITLE
Would External OpenID providers feature be avilable for the free plan users?

### DISCUSSION THREAD

**Discussion by ifoughal** [DefGuard/defguard#770] [Q&A]
Upon reading the [official documentation](https://defguard.gitbook.io/defguard/admin-and-features/external-openid-providers), there is a mention on top regarding the licensing requiring an entreprise plan. Although on the [pricing page](https://defguard.net/pricing/) there is no mention of that, and the pricing only mentions support.

**Comment by teon** [comment]
@ifoughal thank you for noticing - we are in a testing phase of 1.0 which should be released soon (in two weeks?) when also we will publish a new pricing plan (but enterprise without support FYI will start from 10eur/month).

Best way to get notified about all changes is to subscribe: https://defguard.net/newsletter/ (we only send feature updates).
